requirements/base.txt

```diff
@@ -10,6 +10,8 @@ django-celery-beat==2.2.1
 
 sendbox-sdk==0.1.1
 
+django-safedelete==1.3.0
+
 whitenoise==6.2.0
```

Comment:
Why this file have strange structure? Is important empty line? And order of lines? May be make format this file with comment?